Mathlib/Algebra/MonoidAlgebra/Defs.lean

```diff
@@ -807,9 +807,6 @@ protected noncomputable def opRingEquiv [Monoid G] :
       rw [MulOpposite.unop_mul (α := MonoidAlgebra k G), unop_op, unop_op, single_mul_single]
       simp }
 
--- @[simp] -- Porting note (https://github.com/leanprover-community/mathlib4/issues/10618): simp can prove this.
--- More specifically, the LHS simplifies to `Finsupp.single`, which implies there's some
--- defeq abuse going on.
```

Comment:
I think this problem should not just be silenced, since `@[simp]` should not break the `MonoidAlgebra = Finsupp` abstraction barrier. Or is the comment not true anymore?

Reply:
Given that we've chosen to define `abbrev MonoidAlgebra.single := Finsupp.single` using an `abbrev`, the abstraction barrier is pretty weak...

I've removed this porting note in favour of a comment on `abbrev single`.